docs/resources/volume.md

```diff
@@ -88,9 +88,20 @@ resource "ionoscloud_volume" "example" {
   disk_type               = "SSD Standard"
   bus                     = "VIRTIO"
   image_name              = data.ionoscloud_image.example.id
-  image_password          = "K3tTj8G14a3EgKyNeeiY"
+  image_password          = random_password.volume_image_password.result
   user_data               = "foo"
 }
+
+resource "random_password" "server_image_password" {
+  length           = 16
+  special          = false
+}
+
+resource "random_password" "volume_image_password" {
```

Comment:
@gabisavu-ionos the `volume_image_password` isn't the same as the `server_image_password`? I'm looking into the `resource_server.go` and it seems like the `image_password` is the same for both server and volume. Can you please double check this, please? I think that for the `volume_image_password` we should also have `length = 16` and `special = false`.

Reply:
indeed is, I changed it